lib/engine/apps/database.ts

```diff
@@ -142,8 +142,9 @@ export default class AppDatabase extends events.EventEmitter {
         const icon = options.icon || getProgramIcon(program);
         const conversation = options.conversation;
         const notifications = options.notifications;
+        const startTime = new Date;
```

Comment:
You'll probably want to use a milliseconds value here instead of a Date object, because Dates don't survive JSON serialization.

Alternatively, in AppExecutor do `this.startTime = new Date(meta.startTime)` so you know `app.startTime` is a Date object.